packages/legacy/localNodeExec.ts

```diff
@@ -229,15 +224,7 @@ export class CargoLocalNodeExecutor implements LocalNodeExecutor {
    * @param localArgs Local test node commands.
    */
   async cargoRunLocal(localArgs: string[], opts?: { stdin: string }) {
-    const args = [
-      'run',
-      '--bin=blockstack-core',
-      '--package=blockstack-core',
-      '--quiet',
-      '--',
-      'local',
-      ...localArgs
-    ];
+    const args = ['run', '--bin=clarity', '--quiet', '--', ...localArgs];
```

Comment:
We might have a conflict here, since the package `clarity-cli` is also configured for generating an executable name `clarity` (this cli is "user-facing").
Depending on how your PATH is configured, `which clarity` might return the path of cargo package instead of the npm package.

Reply:
I think we should modify blockstack-core repo to output a `clarity-cli` bin file, and have the SDK continue use the name `clarity` for its executable. Since the SDK is intended to be the more convenient thing to use.